mass_editing/hooks.py

```diff
@@ -4,6 +4,5 @@
 
 
 def uninstall_hook(cr, registry):
-    cr.execute("DELETE FROM ir_act_window WHERE res_model = \
-    'mass.editing.wizard'")
-    return True
+    cr.execute("""DELETE FROM ir_act_window WHERE
+    res_model = 'mass.editing.wizard'""")
```

Comment:
you removed return statemnt?

Reply:
Yes sir I removed, I think no need to return statement.